.config/Dockerfile

```diff
@@ -1,6 +1,7 @@
 ARG grafana_version=latest
+ARG grafana_image=grafana-enterprise
```

Comment:
Just asking for myself - is it that we need the enterprise image because it's the recommended default grafana image: https://grafana.com/docs/grafana/latest/setup-grafana/installation/docker/#run-grafana-docker-image?

Reply:
Yep. It's also what `create-plugin` currently uses as a default.